x-pack/test/security_solution_api_integration/test_suites/entity_analytics/default_license/risk_engine/risk_score_calculation.ts

```diff
@@ -132,6 +139,40 @@ export default ({ getService }: FtrProviderContext): void => {
         });
       });
 
+      it('upgrades latest risk score index dynamic before persisting risk scores', async () => {
```

Comment:
I believe this is a decent and fairly simple functional test of the upgrade workflow. However, I bet there are other ways to accomplish this, and am willing to try any alternatives that others might suggest!